source/jormungandr/jormungandr/scenarios/ridesharing/blablacar.py

```diff
@@ -0,0 +1,262 @@
+            )
+            raise RidesharingServiceError(str(e))
+
+    def _make_response(self, raw_json):
+
+        if not raw_json:
+            return []
+
+        ridesharing_journeys = []
+
+        for offer in raw_json:
+
+            res = rsj.RidesharingJourney()
+
+            res.metadata = self.journey_metadata
+            res.distance = offer.get('distance')
+            res.duration = offer.get('duration')
+            res.ridesharing_ad = offer.get('web_url')
+
+            res.pickup_place = rsj.Place(
+                addr='', lat=offer.get('pickup_latitude'), lon=offer.get('pickup_longitude')
+            )
+            res.dropoff_place = rsj.Place(
+                addr='', lat=offer.get('dropoff_latitude'), lon=offer.get('dropoff_longitude')
+            )
+
+            # shape is a list of type_pb2.GeographicalCoord()
+            res.shape = []
+            shape = decode_polyline(offer.get('journey_polyline'), precision=5)
+            if not shape or res.pickup_place.lon != shape[0][0] or res.pickup_place.lat != shape[0][1]:
+                coord = type_pb2.GeographicalCoord()
+                coord.lon = res.pickup_place.lon
+                coord.lat = res.pickup_place.lat
+                res.shape.append(coord)
+            for c in shape:
+                coord = type_pb2.GeographicalCoord()
+                coord.lon = c[0]
+                coord.lat = c[1]
+                res.shape.append(coord)
+            if not shape or res.dropoff_place.lon != shape[0][0] or res.dropoff_place.lat != shape[0][1]:
+                coord = type_pb2.GeographicalCoord()
+                coord.lon = res.dropoff_place.lon
+                coord.lat = res.dropoff_place.lat
+                res.shape.append(coord)
+
+            res.price = offer.get('price').get('amount')
+            if offer.get('price').get('currency') == "EUR":
+                res.currency = "centime"
+            else:
+                res.currency = offer.get('price').get('currency')
+
+            res.available_seats = offer.get('available_seats')
+            res.total_seats = None
+
+            # not specified
+            res.pickup_date_time = None
+            res.dropoff_date_time = None
+
+            res.driver = rsj.Individual(alias=None, gender=None, image=None, rate=None, rate_count=None)
+
+            ridesharing_journeys.append(res)
+
+        return ridesharing_journeys
+
+    def _request_journeys(self, from_coord, to_coord, period_extremity, limit=None):
+        """
+
+        :param from_coord: lat,lon ex: '48.109377,-1.682103'
+        :param to_coord: lat,lon ex: '48.020335,-1.743929'
+        :param period_extremity: a tuple of [timestamp(utc), clockwise]
+        :param limit: optional
+        :return:
+        """
+
+        from_coords = from_coord.split(',')
+        to_coords = to_coord.split(',')
+
+        dt = datetime.datetime.now()
```

Comment:
Why not use _current_datetime?

Reply:
because blablacar need a local time :hear_no_evil: ...